pkg/kubectl/cmd/util/env/env_parse.go

```diff
@@ -28,7 +28,7 @@ import (
 	"k8s.io/kubernetes/pkg/api"
 )
 
-// Env returns an environment variable or a default value if not specified.
+// Env returns an environment variable if defined, or a default value.
```

Comment:
I don't exactly know what I expected this function to do before reading the code, but the `if len(val) == 0` is sligthly dubious. It's probably fine, but it needs to be documented properly.

It's basically the difference between `${parameter-word}` and `${parameter:-word}` in the [shell] world. I'm not sure the current wording (nor the new wording) conveys that semantic properly.

[shell]: http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_02